cmd/kubeadm/app/constants/constants.go

```diff
@@ -377,19 +377,16 @@ var (
 	MasterComponents = []string{KubeAPIServer, KubeControllerManager, KubeScheduler}
 
 	// MinimumControlPlaneVersion specifies the minimum control plane version kubeadm can deploy
-	MinimumControlPlaneVersion = version.MustParseSemantic("v1.12.0")
+	MinimumControlPlaneVersion = version.MustParseSemantic("v1.13.0")
 
 	// MinimumKubeletVersion specifies the minimum version of kubelet which kubeadm supports
-	MinimumKubeletVersion = version.MustParseSemantic("v1.12.0")
+	MinimumKubeletVersion = version.MustParseSemantic("v1.13.0")
 
 	// CurrentKubernetesVersion specifies current Kubernetes version supported by kubeadm
 	CurrentKubernetesVersion = version.MustParseSemantic("v1.13.0")
 
 	// SupportedEtcdVersion lists officially supported etcd versions with corresponding Kubernetes releases
 	SupportedEtcdVersion = map[uint8]string{
-		10: "3.1.12",
-		11: "3.2.18",
-		12: "3.2.24",
```

Comment:
this is basically the support skew - current version minus 2.
having said that i think we should keep 1.12 even if it points to the same version as 1.13.